sdmetrics/reports/multi_table/_properties/inter_table_trends.py

```diff
@@ -135,8 +135,12 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=No
             details['Child Table'] = child
             details['Foreign Key'] = foreign_key
             if not details.empty:
-                details['Column 1'] = details['Column 1'].str.replace(f'{parent}.', '', n=1)
-                details['Column 2'] = details['Column 2'].str.replace(f'{child}.', '', n=1)
+                details['Column 1'] = details['Column 1'].str.replace(
+                    f'{parent}.', '', n=1, regex=False
```

Comment:
Interesting I thought regex=`False` was the default according to the [doc](https://pandas.pydata.org/docs/reference/api/pandas.Series.str.replace.html).

Reply:
It is, but I think Colab is running on an earlier version before it was changed